src/module/rolls/chat-message.ts

```diff
@@ -132,8 +236,7 @@ export class IronswornRollChatMessage {
 
     // Outcome can be overridden
     const theOutcome =
-      this.roll.postAdjustmentOutcome?.value ??
-      this.roll.preAdjustmentOutcome?.value
+      this.roll.finalOutcome?.value ?? this.roll.rawOutcome?.value
     if (!theOutcome) return {}
```

Comment:
Two downsides of using numbers to encode the roll outcome:

1. In Javascript, 0 is falsy. Currently, any move-roll that misses has no move-outcome text.
2. It's harder to read in the debugger:

![CleanShot 2022-08-23 at 08 04 47](https://user-images.githubusercontent.com/39902/186193987-d5db5a95-68a5-4cf7-989f-3cc947ad429f.jpg)

Reply:
oh, good catch! i've added some stricter typing/parsing of numbers/booleans from dialogue options that remedies that.

TBH, i think the `source` being "Roll" there might be a more effective target for revision. in a sense, most things in a roll are sourced from a roll, so it might not be the most effective way of disambiguating the term. further muddying the waters: in the game's parlance, it's still a 'roll' (as in, the mechanical operation that you resolve to an outcome) when the challenge dice come into play, even if you're not actually randomizing any of the dice by rolling them (because they're pre-set, for instance).

so i'd get more specific -- maybe something like "Natural roll outcome" or "Unmodified roll outcome" -- which gives a stronger hint that it's supposed to be a RollOutcome rather than accidentally implying it's a dice value or something. though, the best user-facing label would probably just be "Move outcome" (differs from internal terminology, but it's a more accurate term as far as the game rules are concerned, and that should be a priority for user-facing text), unless the outcome has been modified in some way. otherwise the additional info is just confusing.

so whatever you call it, that item's title might be better handled by the HBS template alone, TBH (and, yes, this is exactly the sort of thing i meant when i expressed misgivings about conflating user-facing labels and internal labels into the same string field ;) )

there's some other implications it has on roll output, but i'll write those up and put them in #366